assets/js/app.js

```diff
@@ -18,3 +18,6 @@ $(function() {
   scrollAdjust();
   navbarBurger();
 });
+
+function externalLinks() { 
```

Comment:
There's a much easier way to do this using Hugo. I'll follow up after this gets merged.